ticket_migrate.py

```diff
@@ -0,0 +1,829 @@
+    status_labels = labels_from_status_and_resolution(status, resolution)
+    labels = (
+        {priority_label}.union(
+            keyword_labels).union(
+            component_labels).union(
+            status_labels)
+        )
+    return sorted(labels)
+
+
+def get_assignees(owner):
+    """
+    Map the owner to the GitHub account.
+    """
+    try:
+        owner, _ = config.USER_MAPPING.get(owner)
+        return [owner]
+    except TypeError as error:
+        if 'cannot unpack non-iterable NoneType object' in str(error):
+            return []
+        raise
+
+
+def showtime(unix_usec):
+    """
+    Convert a Trac timestamp to a human-readable date and time.
+
+    Trac stores timestamps as microseconds since Epoch.
+    """
+    timestamp = unix_usec // 1_000_000
+    dt = datetime.datetime.utcfromtimestamp(timestamp)
+    return f"{dt.isoformat(sep=' ')}Z"
+
+
+def labels_from_component(component: str):
+    """
+    Given the Trac component,
+    choose the labels to apply on the GitHub issue, if any.
+    """
+    if component in config.REPOSITORY_MAPPING:
+        return []
+
+    return [component]
+
+
+def labels_from_keywords(keywords: Union[str, None]):
+    """
+    Given the Trac `keywords` string, clean it up and parse it into a list.
+    """
+    if keywords is None:
+        return set()
+
+    keywords = keywords.replace(',', '')
+    keywords = keywords.split(' ')
+
+    allowed_keyword_labels = {
+        'design',
+        'easy',
+        'feature',
+        'happy-hacking',
+        'onboarding',
+        'password-management',
+        'perf-test',
+        'remote-management',
+        'salt',
+        'scp',
+        'security',
+        'tech-debt',
+        'twisted',
+        'ux',
+        }
+    typo_fixes = {'tech-dept': 'tech-debt', 'tech-deb': 'tech-debt'}
+
+    keywords = [typo_fixes.get(kw, kw) for kw in keywords if kw]
+    discarded = [kw for kw in keywords if kw not in allowed_keyword_labels]
+
+    if discarded:
+        print("Warning: discarded keywords:", discarded)
+
+    return {kw for kw in keywords if kw in allowed_keyword_labels}
+
+
+def labels_from_priority(priority):
+    """
+    Interpret None (missing) priority as Low.
+    """
+    if priority is None:
+        return 'priority-low'
+    return 'priority-{}'.format(priority.lower())
+
+
+def labels_from_status_and_resolution(status, resolution):
+    """
+    The resolution of a closed ticket is used, if there is one.
+    Otherwise, the status is used, if it is not "assigned", "new", or "closed".
+    """
+
+    if status == 'closed' and resolution:
+        return {resolution}
+
+    if status in ['in_work', 'needs_changes', 'needs_merge', 'needs_review']:
+        return {status}
+
+    return set()
+
+
+def parse_body(description):
+    """
+    Parses text with squiggly-bracketed or backtick-surrounded monospace.
+    Converts the squiggly brackets to backtick brackets.
```

Comment:
```suggestion
    Converts the curly brackets to backtick brackets.
```

First time I hear about squiggly :)

Reply:
:D I did not realize my naming was nonstandard